zerver/models.py

```diff
@@ -379,6 +379,12 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
 
     DEFAULT_COMMUNITY_TOPIC_EDITING_LIMIT_SECONDS = 259200
 
+    DEFAULT_MOVE_MESSAGE_LIMIT_SECONDS = 604800
```

Comment:
We should spell the math out so it's clear this is 7 days -- 86400 * 7

Reply:
It might even be worth having a constant at the time of the file `SECONDS_PER_DAY = 86400` and have logic like this use it.

Reply:
Done.